app/views/layouts/application.slim

```diff
@@ -33,3 +33,9 @@ html lang="en"
     = javascript_include_tag :application
     - if Rails.env.production?
       = javascript_include_tag "//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-55ad6e555f728b0f", async: true
+    - unless Rails.env.test?
```

Comment:
~~Let's continue this discussion in https://github.com/subvisual/subvisual.co/pull/117#discussion_r47278176 and I'll resonate the results here afterwards.~~

In fact, just realised I was comparing a Rails app with a Middleman site and that's stupid, apologies. But following that logic, maybe we should then just have Intercom in production. I'll change it for now.